transaction-relayer/src/main.rs

```diff
@@ -268,29 +270,54 @@ fn get_sockets(args: &Args) -> Sockets {
         assert!(!tpu_fwd_ports.contains(&tpu_port));
     }
 
-    let (tpu_p, tpu_quic_sockets): (Vec<_>, Vec<_>) = (0..args.num_tpu_quic_servers)
+    let (tpu_p, tpu_quic_sockets): (Vec<_>, Vec<Vec<_>>) = (0..args.num_tpu_quic_servers)
         .map(|i| {
             let (port, mut sock) = multi_bind_in_range(
                 IpAddr::V4(Ipv4Addr::from([0, 0, 0, 0])),
-                (tpu_ports.start + i, tpu_ports.start + 1 + i),
+                (tpu_ports.start + i, tpu_ports.start + i + 1),
                 1,
             )
             .unwrap();
 
-            (port, sock.pop().unwrap())
+            let quic_config = SocketConfig {
+                reuseaddr: false,
+                reuseport: true,
+            };
+
+            let transactions_quic_sockets = bind_more_with_config(
+                sock.pop().unwrap(),
```

Comment:
does this add to transactions_quic_sockets?

Reply:
it takes `sock` created by `multi_bind_in_range` and returns a vec with that socket plus the others.  it's a little confusing, because `sock` is already a vec of length 1.  but, it just pops off that one socket and binds more on the same ip/port.  

i'm assuming this is just tech debt.  since it would make more sense to replace these 2 function calls with a single multi_bind function.